<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyConsole</class>
 <widget class="QWidget" name="PyConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyConsole</string>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>

   <item>
    <layout class="QHBoxLayout" name="header_layout1">
     <property name="spacing">
      <number>0</number>
     </property>

     <item>
      <widget class="QPushButton" name="load_button">
       <property name="toolTip">
        <string>Load Script</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>

     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="save_button">
       <property name="toolTip">
        <string>Save Script</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="save_as_button">
       <property name="toolTip">
        <string>Save Script As</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>

     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>

    </layout>
   </item>


   <item>
    <layout class="QHBoxLayout" name="header_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="PyConsoleTextEdit" name="py_console"/>
     </item>
    </layout>
   </item>


  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyConsoleTextEdit</class>
   <extends>QTextEdit</extends>
   <header>rqt_py_console.py_console_text_edit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
